nixos/modules/services/misc/home-assistant.nix

```diff
@@ -121,6 +120,19 @@ in {
         ReadWritePaths = "${cfg.configDir}";
         PrivateTmp = true;
       };
+      restartTriggers = [
+        configFile
```

Comment:
Good catch!

Reply:
Actually, thinking about it, shouldn't this be unnecessary becasue `configFile` is mentioned in `preStart`?

Reply:
Yes. It should be unnecessary, since `preStart` is regenerated if `cfg.config` changes.